modules/programs/zsh.nix

```diff
@@ -256,6 +262,8 @@ in
       home.file."${relToDotDir ".zshrc"}".text = ''
         fpath+="$HOME/.nix-profile/share/zsh/site-functions"
         fpath+="$HOME/.nix-profile/share/zsh/$ZSH_VERSION/functions"
+        fpath+="/run/current-system/sw/share/zsh/site-functions"
+        fpath+="/run/current-system/sw/share/zsh/$ZSH_VERSION/functions"
```

Comment:
Would these work for a non-NixOS system?

I note that the zsh module in NixOS [includes](https://github.com/NixOS/nixpkgs/blob/931a0b8be80661902baefb3e7d55403be893e0e6/nixos/modules/programs/zsh/zsh.nix#L162-L165) something like this already.